observatory-dags/observatory/dags/telescopes/google_books.py

```diff
@@ -107,9 +111,19 @@ def transform(self):
                 for row in csv_reader:
                     transformed_row = OrderedDict((convert(k.replace('%', 'Perc')), v) for k, v in row.items())
                     if 'sales' in file:
+                        transaction_date = datetime.strptime(transformed_row['Transaction_Date'], '%m/%d/%Y')
```

Comment:
This format string should be `%-d/%-m/%-y` to match the new date format that Google Books is using, e.g. 1/1/18

Reply:
Ah I see. I updated the test file to match the non-zero date formats. However, the month is before the day, so then it would be `%-m/%-d%/%-y`.
This does not work on mac OS though, I tested with `%m/%d/%y` and that does seem to pick up the 1 decimal months/days correctly.

Reply:
Ah right, that makes sense